src/types/api.types.ts

```diff
@@ -24,6 +24,7 @@ const metaDataSchema = z.object({
   test_takers_count: z.number().int().optional(),
   test_type: z.string().optional(),
   subject: z.string().optional(),
+  status: z.enum(['success', 'pending', 'failed']).optional(),
```

Comment:
for old quizzes.. status would be empty?

Reply:
if session id is empty -- u can assume status is fail. if session id not empty.. u can assume success/?

Reply:
I have handled the Status for old sessions also.
basically I have set it to success by default on UI.